jsonschema-maven-plugin/pom.xml

```
@@ -89,6 +89,11 @@
             <artifactId>jsonschema-module-javax-validation</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.reflections</groupId>
+            <artifactId>reflections</artifactId>
+            <version>0.9.11</version>
```

Can we use the latest version (the one without guava) and declare this as property further up?
```xml
<properties>
    ...
    <reflections.version>0.9.12</reflections.version>
</properties>
```
```suggestion
            <version>${reflections.version}</version>
```

---

Done